src/PrestaShopBundle/Install/XmlLoader.php

```diff
@@ -649,6 +649,10 @@ public function createEntityTab($identifier, array $data, array $data_lang)
             $position[$data['id_parent']] = 0;
         }
         $data['position'] = $position[$data['id_parent']]++;
+        $data['icon'] = $data['icon'] ?? '';
+        $data['wording'] = $data['wording'] ?? '';
+        $data['wording_domain'] = $data['wording_domain'] ?? '';
+        $data['route_name'] = $data['route_name'] ?? '';
```

Comment:
I think it might break an upgrade from 1.6

Reply:
:scream: I'm not sure because fixtures are loaded during installation :thinking:

Reply:
I guess we'll see in the nightly tests anyway 😅